storage/elasticsearch/elasticsearch.go

```diff
@@ -101,8 +99,7 @@ type ElasticSearchStorage struct {
 }
 
 func (c *ElasticSearchStorage) StoreFlows(flows []*flow.Flow) error {
-	c.indexer.Start()
-	defer c.indexer.Stop()
+	defer c.indexer.Flush()
```

Comment:
We need a flush call, but it could be implemented in this story :

     * [F.PROBE] implement Agent.Flush() and Analyzer.Flush() ... 
Storage.Flush() for testing purpose




On 02/16/2016 07:07 PM, Sylvain Afchain wrote:
>
> In storage/elasticsearch/elasticsearch.go 
> <https://github.com/redhat-cip/skydive/pull/70#discussion_r52978253>:
>
> > @@ -101,8 +99,7 @@ type ElasticSearchStorage struct {
> >  }
> >
> >  func (c *ElasticSearchStorage) StoreFlows(flows []*flow.Flow) error {
> > -	c.indexer.Start()
> > -	defer c.indexer.Stop()
> > +	defer c.indexer.Flush()
>
> mmm, Flush is called when Stop, so no need to call it. I hesitated to 
> remove the Flush but I thought you wanted to have something 
> synchronous as you used Stop! finally bulk doesn't mean async :). I 
> will remove it anyway
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/redhat-cip/skydive/pull/70/files#r52978253>.
>